data/json/mutations/mutations.json

```diff
@@ -891,7 +921,7 @@
     "points": 2,
     "mixed_effect": true,
     "description": "You have a unique history with stimulants (like coffee or amphetamines).  You can tolerate a lot more of them without overdosing, but if you indulge too much, you start seeing things…",
-    "valid": false
+    "category": [ "SPIDER" ]
```

Comment:
I'm imagining the pictures of spiderwebs on different substances when I read this change.

Reply:
I'll be real, I was too - I needed some boosts to Spider's Threshold weight, and couldn't get that mockumentary video about spiders on substances out of my head.